get_ramda_file.js

```diff
@@ -3,9 +3,7 @@ var fs = require('fs')
 var path = require('path')
 
 module.exports = (child_path) => new Promise ((resolve, reject) => {
-  var ramda_module_path = require.resolve('ramda')
```

Comment:
ramda resolves to ./src now, which is not what we want here. `var ramda_folder = path.resolve('./node_modules/ramda');` ok?